deployment/instascale-clusterrole.yaml

```diff
@@ -50,3 +50,20 @@ rules:
   - update 
   - delete
   - patch
+
+- apiGroups:
+  - ""
```

Comment:
I think that's an acceptable solution to fix the issue in the short term. However, I think the management of the OCM secret should be reviewed. I think it should be management by Kustomize and possibly mounted into the controller deployment. In the CodeFlare operator, it should be exposed in the InstaScale API as an object reference. If that makes sense, could you please open an issue?

Reply:
Done. https://github.com/project-codeflare/instascale/issues/87